### PR TITLE
Add controller component

### PR DESCRIPTION
**Pull request by phamilton**

### CODE REVIEW

File: components/controller/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Knative-Kafka-Controller 
+
+The knative-kafka-controller application is the Kubernetes Controller implementation for both the Knative Channel
+and ClusterChannelProvisioner CustomResources.  The implementation includes two "Controllers" - one for the 
+ClusterChannelProvisioner and one for the actual Channel instances.  The ClusterChannelProvisioner Controller does
```
Comment (tminke) [inline components/controller/README.md]: I though we'd removed the CCP implementation in which case we should refactor this paragraph ?

File: components/controller/local-env.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+# Absolute Path Of Directory Containing Script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+# Set Local Testing Environment Variables
+export SYSTEM_NAMESPACE="knative-eventing"
+export METRICS_PORT="8081"
+export KAFKA_PROVIDER="azure"
+export KAFKA_BROKERS="eventhub.servicebus.windows.net:9093"
+export KAFKA_OFFSET_COMMIT_MESSAGE_COUNT="50"
```
Comment (tminke) [inline components/controller/local-env.sh:14]: is the kafkabrokers string safe here?  (meaning - is "eventhub" a dummy string?)

**Review by tminke — CHANGES_REQUESTED** [review]
looks good (what i checked) - one minor change and one question

**Review by davyodom — APPROVED** [review]

File: components/controller/local-env.sh (hunk 2 above)
Comment (phamilton) [reply components/controller/local-env.sh:14]: yeah it was a dummy string

File: components/controller/README.md (hunk 1 above)
Comment (phamilton) [reply components/controller/README.md]: Good catch, changed the readme.

**Pull request merged**